.github/workflows/deploy-apis-to-production.yml

```diff
@@ -7,6 +7,11 @@ on:
     types:
       - closed
 
+env:
+  REGISTRY_URL: gcr.io
+  PROJECT_ID: airqo-250220
+  DEPLOY_BRANCH: deployment/argocd
```

Comment:
[use environment variables for container registry](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/0618ee39f85fdbef08c48cd2d26515a4a476b929)